elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -1260,6 +1260,7 @@ elytron.key-manager.credential-reference.type=The type of credential this refere
 elytron.key-manager.credential-reference.clear-text=The secret specified using clear text. Check credential store way of supplying credential/secrets to services.
 elytron.key-manager.key-store=Reference to the KeyStore to use to initialise the underlying KeyManagerFactory.
 elytron.key-manager.alias-filter=A filter to apply to the aliases returned from the KeyStore, can either be a comma separated list of aliases to return or one of the following formats ALL:-alias1:-alias2, NONE:+alias1:+alias2
+elytron.key-manager.generate-self-signed-certificate-host=If the file that backs the KeyStore does not exist and this attribute is set, then a self-signed certificate will be generated for the specified host name. This is not intended for production use.
```

Comment:
How about if the file exists but is empty?

Reply:
The legacy equivalent only generates the self-signed certificate if the file that backs the keystore does not exist, i.e., if the file exists but is empty, legacy security doesn't generate the certificate. Currently, the behaviour with Elytron is the same as it was for legacy security. However, we could change the behaviour with Elytron so that we also generate the certificate for the case where the file exists but is empty.